### ISSUE TITLE
Light wallet stutters every few seconds

### ISSUE THREAD

**Issue by huetsch** [HyperspaceApp/Hyperspace.app#8]
0.2.2, Windows

From el_penor: "afternoon all, when running the HSC wallet, i get stuttering on my machine which is extremely alarming considering it's not a low-end machine. anyone else had such issues? i'm yet to try again with admin priv's."

Stuttering persists across application restarts.

**Comment by huetsch** [comment]
el_penor indicates that giving hsd and hsc admin permissions eliminated the lagging.

**Comment by MuhammadMcgee** [comment]
That's true, problem exist with every sia wallet except maybe SiaPrime. PC almost hangs on start of app, at least the mouse cursor freezes (or become jerky), hard drive activity is very high at this moment, activity led is lit continiously. I, personally, become tempted to press reset button, because it looks like a common hang. But if I wait long enough (around couple of minutes), and let UI load, then it's all fine.
I have a mechanical hard drive thought... dunno about ssd in this scenario.
